src/gui/render/graphics_shaders/individual_int_renderer.hpp

```diff
@@ -0,0 +1,406 @@
+// -*- lsst-c++ -*-
+/*
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ */
+
+/**
+ * @file individual_int_renderer.hpp
+ *
+ * @brief Defines MainRender class.
+ *
+ * @ingroup GUI  MODELS
+ *
+ */
+#pragma once
+
+#include "../../../util/files.hpp"
+#include "../../handler.hpp"
+#include "../../scene/controls.hpp"
+#include "../../shader.hpp"
+#include "../graphics_data/individual_int.hpp"
+#include "gui_render_types.hpp"
+
+#include <GL/glew.h>
+#include <GLFW/glfw3.h>
+#include <glm/glm.hpp>
+#include <glm/gtc/matrix_transform.hpp>
+
+#include <memory>
+#include <vector>
+
+namespace gui {
+
+namespace models {
+
+/**
+ * @brief Renders the meshes to the screen
+ *
+ * @details IndividualIntRenderer renders the meshes given to the screen.
+ * This class handles IndividualInt data structures and classes that derive from
+ * IndividualInt.
+ */
+template <class T>
+class IndividualIntRenderer :
+    public render_to::frame_buffer_multisample,
+    public render_to::frame_buffer,
+    public render_to::shadow_map {
```

Comment:
no no no no no

if you are using these as interfaces put "Interface" in their name.

Reply:
you didn't need five "no"s just to say put interface in the name